collect_app/src/main/java/org/odk/collect/android/activities/GeoPolyActivity.java

```diff
@@ -29,6 +27,8 @@
 import android.widget.Spinner;
 import android.widget.TextView;
 
+import androidx.annotation.VisibleForTesting;
```

Comment:
This got reordered automatically by Android Studio because it went from an `android.` package to an `androidx.` package.